Sources/Abound/TaxProfile.swift

```diff
@@ -36,7 +36,7 @@ public struct TaxProfile: View {
             accessToken: Abound.accessToken,
             theme: theme,
             customContent: customContent,
-            year: "2022",
+            year: "",
```

Comment:
I added this line. This argument was removed by Abound but it's required for `WebView`. There's a compilation error if `year` is not included. The tax profile doesn't use the `year` parameter at all so the value doesn't matter. It would be best if the code were refactored so `year` isn't required at all but that seemed like too much work for now.